### ISSUE TITLE
Incorrect error?

### ISSUE THREAD

**Issue by daurnimator** [stevedonovan/lglob#2]
I thought I'd try and run lglob over [prosody](https://hg.prosody.im/trunk/file/).
Among others, it spat out this error:
`lglob: ./core/certmanager.lua:146: undefined get core_defaults.add_handler`
Which is odd, as the line of code is 
`prosody.events.add_handler("config-reloaded", reload_ssl_config);`

**Comment by stevedonovan** [comment]
Yes - the joy of interpreting assembly dumps and working out exactly what happened ;)

It saw a GETTABLE "add_handler" and mistakenly associated it with the string "config-loaded".

But thanks - this is exactly why I wanted to release this tool, so we can build up a database of patterns that are misunderstood like this.
